pwndbg/hexdump.py

```diff
@@ -12,40 +12,44 @@
 import pwndbg.color
 
 
+color_scheme = None
+printable = None
+
 def groupby(array, count, fill=None):
     array = copy.copy(array)
     while fill and len(array) % count:
         array.append(fill)
     for i in range(0, len(array), count):
         yield array[i:i+count]
 
-#
-# We want to colorize the hex characters
-#
-color_scheme = {i:pwndbg.color.normal("%02x" % i) for i in range(256)}
-
-for c in bytearray((string.ascii_letters + string.digits + string.punctuation).encode('utf-8', 'ignore')):
-    color_scheme[c] = pwndbg.color.bold("%02x" % c)
-
-for c in bytearray(b'\x00\xff'):
-    color_scheme[c] = pwndbg.color.red("%02x" % c)
-
-for c in bytearray(b'\xff\x7f\x80'):
-    color_scheme[c] = pwndbg.color.yellow("%02x" % c)
+def load_color_scheme():
+    global color_scheme, printable
+    #
+    # We want to colorize the hex characters and only print out
+    # printable values on the righ hand side.
+    #
+    color_scheme = {i:pwndbg.color.hexdump_normal("%02x" % i) for i in range(256)}
+    printable = {i:pwndbg.color.hexdump_normal('.') for i in range(256)}
 
-color_scheme[-1] = '  '
+    for c in bytearray((string.ascii_letters + string.digits + string.punctuation).encode('utf-8', 'ignore')):
+        color_scheme[c] = pwndbg.color.hexdump_printable("%02x" % c)
+        printable[c] = pwndbg.color.hexdump_printable("%s" % chr(c))
 
-#
-# Only print out printable values on the righ hand side
-#
-printable = {i:'.' for i in range(256)}
-for c in bytearray((string.ascii_letters + string.digits + string.punctuation).encode('utf-8', 'ignore')):
-    printable[c] = chr(c)
+    for c in bytearray(b'\x00'):
+        color_scheme[c] = pwndbg.color.hexdump_zero("%02x" % c)
+        printable[c] = pwndbg.color.hexdump_zero('.')
 
-printable[-1] = ' '
+    for c in bytearray(b'\xff\x7f\x80'):
+        color_scheme[c] = pwndbg.color.hexdump_special("%02x" % c)
+        printable[c] = pwndbg.color.hexdump_special('.')
 
+    color_scheme[-1] = '  '
+    printable[-1] = ' '
 
 def hexdump(data, address = 0, width = 16, skip = True):
+    global color_scheme, printable
+    if not color_scheme or not printable:
```

Comment:
I'd prefer to just call `load_color_scheme()` once on load.

Separately, since we do not modify `color_scheme` or `printable`, there's no need to declare them as `global`.